examples/step-49/doc/intro.dox

```diff
@@ -99,12 +99,225 @@ many cases it is best to combine both approaches. The file can be easily
+at the bottom of our .geo file to keep the changes when creating our
+<code>.msh</code> file.
+
+@code
+Mesh.Algorithm = 3;
+Mesh.RecombineAll = 1;
+Mesh.CharacteristicLengthFactor = .6;
+Mesh.SubdivisionAlgorithm = 1;
+Mesh.Smoothing = 20;
+Show "*";
+@endcode
+
+To view what each of these do, press <code>Tools -> Options -> Mesh</code>.
+
+You may change the algorithm used for meshing in <code>2D algorithm</code>.
+
+<code>CharacteristicLengthFactor</code> can be thought of the distance between
+nodes on the boundaries of the mesh.  This sets the initial size of the nodes
+of the mesh.  If we want a finer mesh we want a smaller characteristic length.
+For this examples purposes we would like to do more refinement in deal.ii so
+we pick a relatively coarse mesh.  This can also be adjusted in the Options
+menu.
+
+<code>SubdivisionAlgorithm</code> is set to 1 in our file because we want to
+use "All Quads" for subdivision.  In the options menu the default is "None",
+but we use "All Quads" for reasons mentioned earlier.
+
+<code>Smoothing</code> is a post-processing step that iteratively improves mesh quality (for example by moving inner vertices). The number of steps is not crucial in this example.
+For our example we pick 20, however this value can be adjusted
+accordingly, as it's not the most important step in mesh building.
+
+
+
+@note That ends the code for <code>example.geo</code>.  It's important to note that
+  the <code>.geo</code> file did not come out as organized as it appears here.  It
+  was edited after creating the mesh and geometry to be more readable.
+
+Before loading this into deal.ii, we need to create the <code>.msh</code>
+file for deal.II.  So if everything was built
+correctly, then we would like to press <code>Mesh -> 2D</code>.  in Gmsh we
+have to compile the 1D mesh before the 2D mesh.  If all this goes well, then we
+should see something similar to the following output:
+
+<img src="https://www.dealii.org/images/steps/developer/step-49.gmsh_picture_2Dmesh.png" alt="">
+
+It behooves us at this point to also make sure our boundaries were declared
+correctly.  So in the Options menu go to <code>Mesh -> Visibility</code>.
+In the dropdown menu labeled "Label type" we may choose a label for our boundaries.
+We wish to check physical IDs, so we would like to have "Physical tag" picked.
+Now we check "1D element labels" and we should see the numbers assigned to the
+specified boundaries.
+
+<img src="https://www.dealii.org/images/steps/developer/step-49.gmsh_picture_2Dmesh_ids.png" alt="">
+
+deal.II's GridIn class can read the <code>.msh</code> format written by
+%Gmsh and that contains a mesh created for the geometry described by the
+<code>.geo</code> file. You generate the <code>.msh</code> from the
+<code>.geo</code> by running the commands
+
+@code
+gmsh -2 example.geo
+@endcode
+
+on the command line, or by clicking "Mesh" and then "2D" inside %Gmsh after
+loading the file.
```

Comment:
```suggestion
```

Reply:
I don't see a suggestion thing in green so I just assumed that you wanted me to delete it.  If not I can go back and change it again.